cmake/library.cmake

```diff
@@ -44,6 +44,20 @@ function(add_library_target target directory)
     include(${_SOURCE_DIR}/library.cmake)
   endif()
 
+  foreach(_HEADER ${${directory}_HEADERS})
```

Comment:
If anyone knows how to avoid duplicating this CMake nonsense (whose existing form applies only to _subdirectories_), please advise.